src/adcm_pytest_plugin/fixtures.py

```diff
@@ -208,6 +210,27 @@ def _attach_adcm_logs(request: SubRequest, adcm: ADCM):
             )
 
 
+def _attach_adcm_url(request: SubRequest, adcm: ADCM):
+    """Attach ADCM URL link to the Allure Report for the further access"""
+    attachment_name = "ADCM API URL"
```

Comment:
Most likely you meant the link to the admin panel and not API?

Reply:
Actually it is something like `http://{host}:{port}`

Reply:
I suggest leaving just `ADCM URL` without `API`

Reply:
Done